03week/pigLatinGUI.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+console.log("Script working");
+
+let userInput = document.querySelector("#userInput");
+let inputText = userInput.value;
+let translateButton = document.querySelector("#translateButton");
+let translationOutput = document.querySelector('#output');
+console.log(inputText);
+//Pig latin translation script
+const pigLatin = (plWord)  => {
+  //Declaring variables to split the word, and concat into new variable
+  let firstPart;
+  let secondPart;
+  let newWord;
+  let scrubWord = plWord.toLowerCase().split("");
+  //Function that finds the first vowel
+  const isVowel = (word) => {
+    for (let i=0;i<word.length;i++) {
+      if (word[0] === "a" || word[0] === "e" || 
+      word[0] === "i" || word[0] === "o" || word[0] === "u") {
```

Comment:
its odd that this is inside the loop, since you are only checking the first letter.
I would do that check outside the loop, and only go in the loop if the first letter is not a vowel